python/python/tests/test_pubsub.py

```diff
@@ -205,11 +273,18 @@ async def test_pubsub_exact_happy_path(
 
             await check_no_messages_left(method, listening_client, callback_messages, 1)
         finally:
-            if cluster_mode:
-                # Since all tests run on the same cluster, when closing the client, garbage collector can be called after another test will start running
-                # In cluster mode, we check how many subscriptions received the message
-                # So to avoid flakiness, we make sure to unsubscribe from the channels
-                await listening_client.custom_command(["UNSUBSCRIBE", channel])
+            # if cluster_mode:
+            #     # Since all tests run on the same cluster, when closing the client, garbage collector can be called after another test will start running
+            #     # In cluster mode, we check how many subscriptions received the message
+            #     # So to avoid flakiness, we make sure to unsubscribe from the channels
+            #     await listening_client.custom_command(["UNSUBSCRIBE", channel])
```

Comment:
remove?

Reply:
no, i need the old cleanup code here for now